tests/renderer/components/settings-spec.tsx

```diff
@@ -118,15 +118,15 @@ describe('Settings component', () => {
 
     // trigger mock 'keyup' event
     map.keyup({ code: 'Escape' });
-    expect(Object.keys(map)).toHaveLength(1);
+    expect(Object.keys(map)).toHaveLength(2); // {'keyup','contextmenu'}
     expect(store.isSettingsShowing).toBe(true);
 
     // Toggle the setting again as if it was closed
     instance.toggleHasPopoverOpen();
 
     // trigger mock 'keyup' event
     map.keyup({ code: 'Escape' });
-    expect(Object.keys(map)).toHaveLength(1);
+    expect(Object.keys(map)).toHaveLength(2); // {'keyup','contextmenu'}
     expect(store.isSettingsShowing).toBe(false);
 
```

Comment:
```suggestion

    // trigger mock 'contextmenu' event
    const preventDefault = jest.fn();
    map.contextmenu({ preventDefault });
    expect(Object.keys(map)).toHaveLength(2); // {'keyup','contextmenu'}
    expect(preventDefault).toHaveBeenCalled();

```

I haven't tested it, but you could try that to get a bit of test coverage. 🙂